test/Window/WindowBase.test.cpp

```diff
@@ -29,6 +29,27 @@ TEST_CASE("[Window] sf::WindowBase" * doctest::skip(skipDisplayTests))
             CHECK(windowBase.getSystemHandle() == sf::WindowHandle());
         }
 
+        SUBCASE("Move semantics")
+        {
+            SUBCASE("Construction")
+            {
+                sf::WindowBase       movedWindowBase;
+                const sf::WindowBase windowBase(std::move(movedWindowBase));
+                CHECK(windowBase.getSystemHandle() == sf::WindowHandle());
+                CHECK(windowBase.getSize() == sf::Vector2u());
+            }
+
+            SUBCASE("Assignment")
+            {
+                sf::WindowBase windowBase;
+                {
+                    sf::WindowBase movedWindowBase(sf::VideoMode({360, 240}), "WindowBase Tests");
+                    windowBase = std::move(movedWindowBase);
+                }
+                CHECK(windowBase.getSize() == sf::Vector2u(360, 240));
```

Comment:
```suggestion
                sf::WindowBase movedWindowBase(sf::VideoMode({360, 240}), "WindowBase Tests");
                sf::WindowBase windowBase;
                windowBase = std::move(movedWindowBase);
                CHECK(windowBase.getSize() == sf::Vector2u(360, 240));
```
I think it's fine to let the moved-from window stay in scope for the sake of keeping the tests simple.